### PR TITLE
Update Chinese (Simplified) translation

### PR DESCRIPTION
**Pull request by imgradeone**

### CODE REVIEW

File: po/zh_CN.po [hunk 1]
```diff
@@ -8,127 +8,124 @@ msgstr ""
...
+msgstr "视频"
 
 #: src/Panes/AudiencePane.vala:35
 msgid "Stay on top while playing:"
-msgstr ""
+msgstr "播放时置顶："
 
 #: src/Panes/AudiencePane.vala:38
 msgid "Move window from video canvas:"
 msgstr ""
```
Comment (ryonakano) [inline po/zh_CN.po:88]: Would you add translation for this string too?

File: po/zh_CN.po [hunk 2]
```diff
@@ -8,127 +8,124 @@ msgstr ""
...
 
 #: src/Panes/AnimationsPane.vala:41
-#, fuzzy
 msgid "Close:"
-msgstr "仅关闭按钮"
+msgstr "关闭："
 
 #: src/Panes/AnimationsPane.vala:45
 msgid "Snap:"
 msgstr ""
```
Comment (ryonakano) [inline po/zh_CN.po:120]: Here too

**Review by ryonakano — COMMENTED** [review]
Hi, thanks for your translation!

Looks like two translation are missing so I would appreciate if you would add translations for them too.

**Comment by imgradeone** [comment]
Well I'm not quite sure how to translate "Snap".

**Comment by ryonakano** [comment]
As far as googled the word "分屏", I guess this translation seems to be fine.

**Review by ryonakano — APPROVED** [review]
Looks good, 非常谢谢！

**Pull request merged**